CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
+- [#33](https://github.com/zendframework/zend-di/pull/32) add ability to pass
+  `Psr\Log\LoggerInterface` to `Zend\Di\CodeGenerator\InjectorGenerator`'s constructor
+
+
 - [#32](https://github.com/zendframework/zend-di/pull/32) adds the implementation of
```

Comment:
@Ocramius I'm a bit unsure about leaving this in the changelog. Should it be removed as for readability (Same goes for the Removed and Fixed section). Since the changes from PR #32 were never released.

Reply:
Note is fine as long as you aggregate it with the changes in this patch. What matters is that the reader of the release notes understands what happened.

Reply:
I guess it would be more confusing then, since it may lead people to the assumption that 3.1 introduces the Aware implementation which is actually not the case. I moved the reference to PR #32 to the Removed and Fixed entries to state clearly that this is not the case.